lib/loader.js

```diff
@@ -53,6 +41,13 @@ function sassLoader(content) {
         addNormalizedDependency
     ));
 
+    if (asyncSassJobQueue === null) {
+        const implementation = options.implementation || require("node-sass");
+
+        validateSassVersion(implementation.info);
+        asyncSassJobQueue = async.queue(implementation.render, threadPoolSize - 1);
```

Comment:
The async job queue mitigates a problem of node-sass/libsass where the threadpool gets exhausted (see https://github.com/sass/node-sass/issues/857#issuecomment-93594360). Do you know if dart-sass suffers from the same problem? (I don't think so, unless you ask for a thread from libuv like node-sass does)

If not, we could get rid of the queue for dart-sass.

Reply:
I don't believe Dart Sass suffers from that problem, but I wanted to keep the code path as similar between the two implementations as possible.

Reply:
Seems reasonable, but I suspect the queue to be one of the performance bottlenecks as discussed in #296. However, that's not a blocker for this PR.